### PR TITLE
Add constants for F_OFT_* locks

### PR DESCRIPTION
**Pull request by nemanja-boric-sociomantic**
This adds constants for Linux's F_OFD_GETLK, F_OFD_SETLK and
F_OFD_SETLKW.

### CODE REVIEW

File: src/core/sys/linux/fcntl.d [hunk 1]
```diff
@@ -17,6 +17,24 @@ enum {
     FALLOC_FL_UNSHARE_RANGE = 0x40
 }
 
+// From asm-generic/fcntl.h
+/*
```
Comment (wilzbach) [inline src/core/sys/linux/fcntl.d]: Any reason why you didn't copy the title as well: "Open File Description Locks"?

Comment (wilzbach) [inline src/core/sys/linux/fcntl.d]: I see that the other definition here isn't marked with a public Ddoc comment, but I don't see any reason why it shouldn't be exposed in the docs?

Comment (Burgos) [inline src/core/sys/linux/fcntl.d]: Not really, I'll update tomorrow with the ddoc (will fix entire file) and add the title. Thanks!

**Comment by nemanja-boric-sociomantic** [comment]
Updated with the ddoc done on the file.

**Comment by wilzbach** [comment]
> Updated with the ddoc done on the file.

Thanks a lot. Still LGTM - anyone objecting? Feel free to merge this in two days with your @Burgos account if no further words are heard ;-)

**Pull request merged**